openmrs/omod/src/main/webapp/profiles.jsp

```diff
@@ -2,104 +2,155 @@
 <%@ include file="/WEB-INF/template/header.jsp"%>
 <%@ include file="template/localHeader.jsp"%>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>
+<%@ page trimDirectiveWhitespaces="true" %>
 <style>
-  .section { margin: 2em 0; }
-  div.message { padding: 1em; background: #ffe; border: 1px solid #aa5; }
-  div.error.message { background: #fee; border: 1px solid #a55; }
-  .output { background: #ccc; padding: 1em; }
-  .selected { font-weight: bold; }
-  .none { color: #999; }
-  .filename { font-family: monospace; font-size: 16px; }
-  .profile-select { white-space: nowrap; }
-  select { vertical-align: top; width: 50em; }
-  input { font-size: 16px; }
+    .section {
+        margin: 2em 0;
+    }
+    div.message {
+        padding: 1em;
+        background: #ffe;
+        border: 1px solid #aa5;
+    }
+    div.error.message {
+        background: #fee;
+        border: 1px solid #a55;
+    }
+    .output {
+        background: #ccc;
+        padding: 1em;
+    }
+    .selected {
+        font-weight: bold;
+    }
+    .none {
+        color: #999;
+    }
+    .filename {
+        font-family: monospace;
+        font-size: 16px;
+    }
+    .profile-select {
+        white-space: nowrap;
+    }
+    select {
+        vertical-align: top;
+        width: 50em;
+        margin-left: 8px;
+    }
+    input {
+        font-size: 16px;
+    }
+    #upload > span  {
+        position: relative;
+    }
+    #upload > span > input[type="file"] {
+        position: absolute;
+        opacity: 0;
+    }
+    #upload > span > input[type="submit"] {
+        position: absolute;
+        margin-left: 30px;
```

Comment:
If you really want to horizontally align the profile name, the button, and the selection list, don't do it by inserting specific numbers of pixels like this; either use a table, or wrap the labels ("Current profile", "Add a profile", "Select a profile") in their own elements with all the same fixed width, e.g. `display: inline-block; min-width: 10em;`